src/lib.rs

```diff
@@ -13,16 +13,14 @@
 pub mod rpc;
 pub mod utransport;
 
-use protobuf::{Enum, Message};
+use bitmask_enum::bitmask;
+use protobuf::Message;
 use std::{
     collections::HashMap,
     sync::{Arc, Mutex},
 };
 use tokio::runtime::Runtime;
-use up_rust::{
-    ComparableListener, UAttributes, UAuthority, UCode, UEntity, UListener, UPayloadFormat,
-    UPriority, UResourceBuilder, UStatus, UUri,
-};
+use up_rust::{ComparableListener, UAttributes, UCode, UListener, UPriority, UStatus, UUri};
 use zenoh::{
     config::Config,
```

Comment:
Suggest re-exporting certain key things from the Zenoh crate, maybe start with Config?

Reply:
Good idea